src/feedback/trulens/feedback/llm_provider.py

```diff
@@ -309,11 +309,24 @@ def _determine_output_space(
             f"Invalid score range: [{min_score_val}, {max_score_val}]. Must match one of the predefined output spaces."
         )
 
+    def _determine_output_space_prompt(self, output_space_name: str) -> str:
```

Comment:
A bit confused here - does the existing output space validation logic using pydantic validator not work? 

We have a mixin dedicated to this and this is called within `generate_system_prompt`
https://github.com/truera/trulens/blob/main/src/feedback/trulens/feedback/v2/feedback.py#L246

Reply:
Ah okay I saw you removed the logic in `CriteriaOutputSpaceMixin` below. could you elaborate a bit why doing so - my main concern is that now we are repeating `output_space_prompt = self._determine_output_space_prompt(output_space)` in almost every feedback function, and we are also forcing the dev to do the same in the future.

Reply:
Not sure if the validation works, but the output space prompt generation was not working. I moved it so I could call it directly and pass the output space prompt to the system message generator